File: AlveoEnergy/src/app/models/inventory/relay-item.ts
import { ModelBase } from '../model-base';

export class RelayItem extends ModelBase {
    public Name: string;
    public Make: string;
    public PartNumber: string;
    public Qty: number;
    public Description: string;
    public Unit: string;
    public Rate: number;
    public Req: number;
    public Ammount: Number;

    constructor() {
        super();

    }

    ToJson(): string {
        return JSON.stringify(this);
    }

    static FromJson(jObj: any): RelayItem {
        var inventoryItem = new RelayItem();
        inventoryItem.ID = jObj.ID;
        inventoryItem.Name = jObj.Name;
        inventoryItem.PartNumber = jObj.PartNumber;
        inventoryItem.Qty = jObj.Qty;
        inventoryItem.Make = jObj.Make;
        return inventoryItem;
    }

}

File: AlveoManagementServer/AlveoManagementCommon/Classes/Inventory_Classes/RelayItem.cs
using AlveoManagementCommon.Interfaces;

namespace AlveoManagementCommon.Classes
{
    public class RelayItem : IRelayItem
    {
        public string ID { get; set; }
        public string Name { get; set; }
        public string Make { get; set; }
        public string PartNumber { get; set; }
        public int Qty { get; set; }
        public int Rate { get; set; }
        public string Description { get; set; }
        public string Unit { get; set; }
        public int Req { get; set; }
    }
}